src/utils/editor/index.js

```diff
@@ -117,7 +119,7 @@ function traverseResults(e, ctx, query, dir, modifiers) {
 }
 
 function createEditor() {
-  return new SourceEditor({
+  thisEditor = new SourceEditor({
```

Comment:
I save this `SourceEditor` because we need to pass this editor to [getTextForLine](https://github.com/devtools-html/debugger.html/pull/2318/files#diff-695f8e845eb7e7930a1b665650866b59R70) when we dispatch `addBreakpoint`. If we don't call `getTextForLine`, we'll not have text below a breakpoint in the `SecondaryPanes`.

**Without text**
<img width="1223" alt="screen shot 2017-03-10 at 10 59 19 am" src="https://cloud.githubusercontent.com/assets/4562118/23784425/f1e43b2c-059c-11e7-8ecc-ce90d741f370.png">

**With text**
<img width="1214" alt="screen shot 2017-03-10 at 10 59 43 am" src="https://cloud.githubusercontent.com/assets/4562118/23784429/f9998ab6-059c-11e7-9b4e-0a7f8c54061d.png">